modules/gapi/samples/infer_single_roi.cpp

```diff
@@ -164,7 +219,7 @@ int main(int argc, char *argv[])
                   << std::endl;
         cv::GOpaque<cv::Rect> in_roi;
         auto blob = cv::gapi::infer<custom::FaceDetector>(in_roi, in);
-        cv::GArray<cv::Rect> rcs = cv::gapi::parseSSD(blob, sz, 0.5f, true, true);
+        auto rcs = custom::ParseSSD::on(blob, in_roi, sz);
```

Comment:
What is the purpose of creation of new one?
What is the difference between them?
Could we extend existing one with extra mode?

/cc @dmatveev We need your approval here.

Reply:
@alalek, `custom::ParseSSD` upscales ssd result by input roi. Unfortunately `gapi::parseSSD` kernels don't have this functionality.

Reply:
3rd question is left.

Before implementing some functionality in samples only we need to evaluate its value to be a part of related module itself.

Reply:
@alalek running a detector within ROI is mainly a synthetic case, so custom kernel is OK.

However, I'd consider using our "standard" parseSSD/parseYolo functions and introducing a custom kernel which makes just a coordinate shift based on the ROI. @mpashchenkov  could you do it that way?

Reply:
Yes, it will be a simple overload as far as I remember.